src/WinRT.Runtime/ComWrappersSupport.net5.cs

```diff
@@ -83,7 +88,8 @@ public static T CreateRcwForComObject<T>(IntPtr ptr)
             // ComWrappers API surface, so we are achieving it via a thread local.  We unset it after in case
             // there is other calls to it via other means.
             CreateRCWType.Value = typeof(T);
-            var rcw = ComWrappers.GetOrCreateObjectForComInstance(ptr, CreateObjectFlags.TrackerObject);
+
+            var rcw = tryUseCache ? ComWrappers.GetOrCreateObjectForComInstance(ptr, CreateObjectFlags.TrackerObject) : ComWrappers.GetOrCreateObjectForComInstance(ptr, CreateObjectFlags.UniqueInstance);
```

Comment:
We should still or TrackerObject to UniqueInstance in the 2nd case.  Consider making it a variable and do the ternary when setting the variable which is then passed to the call.